app/src/main/res/layout/topic_download_dialog.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:padding="@dimen/cellular_data_dialog_padding">
```

Comment:
We can make use of FrameLayout if it contains only one view.

Reply:
That would actually be the part of Hi-fi and I am currently working on low-fi only.